### ISSUE TITLE
Color contrast of Explore page in light and dark mode

### ISSUE THREAD

**Issue by AyushRaj-10** [Suhani1234-5/TourEase#93]
### Describe the Bug

The color contrast of Explore page is different from other pages in dark and light page.  am under SWOC 2026 and i want to work on this issue, can you assign that to me, please

### Steps to Reproduce

Go to the website and click on Explore button in dark mode

### Expected Behavior

The color contrast should be same as others

**Comment by Pratik-Bavche** [comment]
I can help with this issue, as I have already worked on similar issues before. Kindly assign it to me.

**Comment by anshikaagr** [comment]
@Suhani1234-5  Please assign this issue to me under GSSoC. I would like to work on it.

**Comment by vishakhaojha57** [comment]
@Suhani1234-5 Please assign this issue to me under GSSoC. I would like to work on it.
